src/components/menus/SnippetList.jsx

```diff
@@ -16,11 +16,13 @@ const makeMenuItems = _.memoize((userSnippets, onClick) => {
 });
 
 const SnippetList = ({ titles, onClick }) => {
+  const menuItems = makeMenuItems(titles, onClick);
   return (
     <MenuItem
       primaryText="My Snippets"
       rightIcon={<ArrowDropRight />}
-      menuItems={makeMenuItems(titles, onClick)}
+      disabled={menuItems ? false : true}
```

Comment:
I prefer it because it explicitly sets it to a Boolean value instead of relying on JS' truthiness. Again, it's not a reason to block, just a nitpick I notice.